sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/src/Generated/Models/TriggerStatus.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Azure.Management.ContainerRegistry.Models
     /// </summary>
     public static class TriggerStatus
     {
-        public const string Disabled = "Disabled";
         public const string Enabled = "Enabled";
+        public const string Disabled = "Disabled";
```

Comment:
To me, this means the code-gen I am using might be different. To avoid random change, please let me know what version I should use. Right now I have 
```
C:\sdk\azure-sdk-for-net>autorest --version
AutoRest code generation utility [version: 2.0.4413; node: v10.19.0]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest


Showing All Installed Extensions

 Type       Extension Name                           Version      Location
 core       @microsoft.azure/autorest-core           2.0.4302     C:\Users\yugangw\.autorest\@microsoft.azure_autorest-core@2.0.4302
 core       @microsoft.azure/autorest-core           2.0.4413     C:\Users\yugangw\.autorest\@microsoft.azure_autorest-core@2.0.4413
 extension  @microsoft.azure/autorest.csharp         2.3.82       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.csharp@2.3.82
 extension  @microsoft.azure/autorest.modeler        2.3.44       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.modeler@2.3.44
 extension  @microsoft.azure/autorest.modeler        2.3.55       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.modeler@2.3.55
 extension  @microsoft.azure/autorest.python         3.0.62       C:\Users\yugangw\.autorest\@microsoft.azure_autorest.python@3.0.62

```

Reply:
Looks like the `generate.ps1` installs the autorest version it requires, so I would trust the new code-gen does the right thing
```
+ autorest@2.0.4413
updated 1 package in 0.694s
```

Reply:
Yes, we are forcing autorest to use v2 because v3 doesn't work with track 1 c# generator